R/api.R

```diff
@@ -30,7 +30,7 @@ api_createProtocol <- function(protocolName, shortDescription, recordedBy, stopO
   
   if(stopOnAnyExist) {
     if(any(unlist(protocolExistence))) {
-      stopUser(paste0(racas::applicationSettings$client.protocol.label,"s already registered: '",paste(names(protocolExistence[unlist(protocolExistence)]),collapse = "', '")),"'")
+      stopUser(paste0(racas::applicationSettings$client.protocol.label,"s already registered: '",paste(names(protocolExistence[unlist(protocolExistence)]),collapse = "', '"),"'"))
```

Comment:
Sorry to include this here but R was giving me a warning that this was going to break so I fixed it.